<Type Name="ReturnOutOfFinally" FullName="Microsoft.JScript.ReturnOutOfFinally">
  <Metadata>
    <Meta Name="ms.openlocfilehash" Value="d1284af23b567886e142ffcc1d8b8352e561ffdd" />
    <Meta Name="ms.sourcegitcommit" Value="6a0b904069161bbaec4ffd02aa7d9cf38c61e72e" />
    <Meta Name="ms.translationtype" Value="HT" />
    <Meta Name="ms.contentlocale" Value="it-IT" />
    <Meta Name="ms.lasthandoff" Value="06/24/2018" />
    <Meta Name="ms.locfileid" Value="36661997" />
  </Metadata>
  <TypeSignature Language="C#" Value="public sealed class ReturnOutOfFinally : ApplicationException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable sealed beforefieldinit ReturnOutOfFinally extends System.ApplicationException" />
  <TypeSignature Language="DocId" Value="T:Microsoft.JScript.ReturnOutOfFinally" />
  <TypeSignature Language="VB.NET" Value="Public NotInheritable Class ReturnOutOfFinally&#xA;Inherits ApplicationException" />
  <TypeSignature Language="C++ CLI" Value="public ref class ReturnOutOfFinally sealed : ApplicationException" />
  <TypeSignature Language="F#" Value="type ReturnOutOfFinally = class&#xA;    inherit ApplicationException" />
  <AssemblyInfo>
    <AssemblyName>Microsoft.JScript</AssemblyName>
    <AssemblyVersion>10.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.ApplicationException</BaseTypeName>
  </Base>
  <Interfaces />
  <Docs>
    <summary>
      <span data-ttu-id="b410e-101">Rappresenta lo stato di eccezione quando l'esecuzione del codice torna all'esterno di un blocco <see langword="finally" />.</span>
      <span class="sxs-lookup">
        <span data-stu-id="b410e-101">Represents the exception state when code execution returns out of a <see langword="finally" /> block.</span>
      </span>
    </summary>
    <remarks>To be added.</remarks>
    <forInternalUseOnly />
    <altmember cref="T:Microsoft.JScript.Try" />
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>Microsoft.JScript</AssemblyName>
        <AssemblyVersion>10.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary>
          <span data-ttu-id="b410e-102">Inizializza una nuova istanza della classe <see cref="T:Microsoft.JScript.ReturnOutOfFinally" />.</span>
          <span class="sxs-lookup">
            <span data-stu-id="b410e-102">Initializes a new instance of the <see cref="T:Microsoft.JScript.ReturnOutOfFinally" /> class.</span>
          </span>
        </summary>
        <altmember cref="T:Microsoft.JScript.Try" />
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ReturnOutOfFinally ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:Microsoft.JScript.ReturnOutOfFinally.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ReturnOutOfFinally();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>Microsoft.JScript</AssemblyName>
        <AssemblyVersion>10.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary>
          <span data-ttu-id="b410e-103">Inizializza una nuova istanza della classe <see cref="T:Microsoft.JScript.ReturnOutOfFinally" />.</span>
          <span class="sxs-lookup">
            <span data-stu-id="b410e-103">Initializes a new instance of the <see cref="T:Microsoft.JScript.ReturnOutOfFinally" /> class.</span>
          </span>
        </summary>
        <remarks>To be added.</remarks>
        <forInternalUseOnly />
        <altmember cref="T:Microsoft.JScript.Try" />
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ReturnOutOfFinally (string m);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string m) cil managed" />
      <MemberSignature Language="DocId" Value="M:Microsoft.JScript.ReturnOutOfFinally.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (m As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ReturnOutOfFinally(System::String ^ m);" />
      <MemberSignature Language="F#" Value="new Microsoft.JScript.ReturnOutOfFinally : string -&gt; Microsoft.JScript.ReturnOutOfFinally" Usage="new Microsoft.JScript.ReturnOutOfFinally m" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>Microsoft.JScript</AssemblyName>
        <AssemblyVersion>10.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="m" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="m">
          <span data-ttu-id="b410e-104">Messaggio in cui viene descritto l'errore.</span>
          <span class="sxs-lookup">
            <span data-stu-id="b410e-104">The message that describes the error.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="b410e-105">Inizializza una nuova istanza della classe <see cref="T:Microsoft.JScript.ReturnOutOfFinally" /> utilizzando un messaggio di errore specificato.</span>
          <span class="sxs-lookup">
            <span data-stu-id="b410e-105">Initializes a new instance of the <see cref="T:Microsoft.JScript.ReturnOutOfFinally" /> class, using the specified error message.</span>
          </span>
        </summary>
        <remarks>To be added.</remarks>
        <forInternalUseOnly />
        <altmember cref="T:Microsoft.JScript.Try" />
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ReturnOutOfFinally (string m, Exception e);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string m, class System.Exception e) cil managed" />
      <MemberSignature Language="DocId" Value="M:Microsoft.JScript.ReturnOutOfFinally.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (m As String, e As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ReturnOutOfFinally(System::String ^ m, Exception ^ e);" />
      <MemberSignature Language="F#" Value="new Microsoft.JScript.ReturnOutOfFinally : string * Exception -&gt; Microsoft.JScript.ReturnOutOfFinally" Usage="new Microsoft.JScript.ReturnOutOfFinally (m, e)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>Microsoft.JScript</AssemblyName>
        <AssemblyVersion>10.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="m" Type="System.String" />
        <Parameter Name="e" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="m">
          <span data-ttu-id="b410e-106">Messaggio in cui viene descritto l'errore.</span>
          <span class="sxs-lookup">
            <span data-stu-id="b410e-106">The message that describes the error.</span>
          </span>
        </param>
        <param name="e">
          <span data-ttu-id="b410e-107">Eccezione che ha determinato l'eccezione corrente.</span>
          <span class="sxs-lookup">
            <span data-stu-id="b410e-107">The exception that is the cause of the current exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="b410e-108">Inizializza una nuova istanza della classe <see cref="T:Microsoft.JScript.ReturnOutOfFinally" /> utilizzando il messaggio di errore specificato e un riferimento all'eccezione interna che Ã¨ la causa di questa eccezione.</span>
          <span class="sxs-lookup">
            <span data-stu-id="b410e-108">Initializes a new instance of the <see cref="T:Microsoft.JScript.ReturnOutOfFinally" /> class, using the specified error message and a reference to the inner exception that is the cause of this exception.</span>
          </span>
        </summary>
        <remarks>To be added.</remarks>
        <forInternalUseOnly />
        <altmember cref="T:Microsoft.JScript.Try" />
      </Docs>
    </Member>
  </Members>
</Type>